stores/hostdb.go

```diff
@@ -697,8 +681,8 @@ func (ss *SQLStore) HostBlocklist(ctx context.Context) (blocklist []string, err
 
 // RecordHostInteraction records an interaction with a host. If the host is not in
 // the store, a new entry is created for it.
-func (ss *SQLStore) RecordInteractions(ctx context.Context, interactions []hostdb.Interaction) error {
-	if len(interactions) == 0 {
+func (ss *SQLStore) RecordInteractions(ctx context.Context, scans []hostdb.HostScan, priceTableUpdate []hostdb.PriceTableUpdate) error {
```

Comment:
docstring doesn't match but I'm wondering whether it makes sense to keep calling this "interactions". Should we take the time to split it up and record scans and pt updates separately? I don't really see why those should be grouped now that we got rid of the interaction type